source/wp-content/plugins/wporg-post-title/block.json

```diff
@@ -0,0 +1,49 @@
+{
+	"apiVersion": 2,
+	"name": "wporg/post-title",
+	"title": "WordPress.org Post Title",
+	"category": "theme",
+	"description": "Extends the core Post Title block with additional attributes needed for WordPress.org themes.",
```

Comment:
Aside from the `releaseVersion` toggle, how is this different from the Post Title block? Only asking because when I saw "custom block for the version" I thought of making a simple block that _only_ outputs the release version, with no other attributes.

Reply:
Yeah I like that idea. I guess I was preempting other hypothetical uses for the same block, but I don't think we have any other actual use cases right now.